### DISCUSSION TITLE
My recovery codes don't work

### DISCUSSION THREAD

**Discussion by AdrCard90** [community/community#146691] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I haven't used my work GitHub account for a long time but i have stored my recovery codes in case i need them. Today i am trying to recover my password and i don't have the Authenticator app, so i initiated the recovery using recovery codes, however none of them worked, i tried refreshing to have new codes but even the new ones didn't work... I have no idea what to do now.

**Comment by pooulad** [comment]
### Use GitHub's account recovery process

1. **Visit GitHub's account recovery page**: Go to [GitHub account recovery](https://github.com/login/recovery) and start the process.
2. **Fill out the form**: Provide any requested details such as your email address, the username, or any other information that can help identify you.
3. **Wait for GitHub's support team**: After submitting the form, GitHub will review your case and get in touch with you. They may ask for additional information to verify your identity.
